source/lib.ts

```diff
@@ -168,11 +176,23 @@ const parseOptions = (passedOptions: Partial<Options>): Configuration => {
 	const notUndefinedOptions: Partial<Options> =
 		omitUndefinedOptions(passedOptions)
 
-	// Create the validator before even parsing the rest of the options
+	// Create the validator before even parsing the rest of the options.
 	const validations = new Validations(notUndefinedOptions?.validate ?? true)
 
+	// Warn for the deprecated option.
 	validations.onLimitReached(notUndefinedOptions.onLimitReached)
 
+	// The default value for the `standardHeaders` option is `false`. If it is set
+	// to `true`, resolve it as `draft-7`. Note that the recommended value is `draft-7`.
```

Comment:
`true` should be treated as `draft-6` forbackwards compatibility:

```suggestion
	// The default value for the `standardHeaders` option is `false`. 
	// If set to `true`, it resolve to `draft-6`. 
	// The deprecated `draft_polli_ratelimit_headers` option also resolves to `draft-6`.
	// `draft-7` (recommended) is used only if explicitly set.
```